src/pages/landing/ebooks/components/_topics.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
 import styled from 'styled-components'
-import PropTypes from 'prop-types'
+import { ImageDataLike } from 'gatsby-plugin-image'
```

Comment:
```suggestion
import type { ImageDataLike } from 'gatsby-plugin-image'
```
we will need this as Gatsby had explicitly exported it as a type declaration.

Reply:
Nice